### PR TITLE
Add page description

### PR DESCRIPTION
**Pull request by inductor**
Current OGP rendering is not very nice. So here's some improvement.
I only speak Japanese but not Chinese(I understand many of the letters though) so please let me know if you want to improve the wording.

image

### CODE REVIEW

**Review by jim3ma — APPROVED** [review]
LGTM

File: docsite.config.yml [hunk 1]
```diff
@@ -5,31 +5,31 @@ pages:
...
    keywords: 'keyword1,keyword2'
-   description: 'page description'
+   description: 'An Open-source P2P-based Image and File Distribution System'
  community:
   # 社区页配置
   zh-cn:
    title: '社区'
    keywords: '关键词1，关键词2'
-   description: '页面内容简介'
+   description: 'Dragonfly的社区'
```
Comment (jim3ma) [inline docsite.config.yml:19]: Dragonfly社区

File: docsite.config.yml [hunk 2]
```diff
@@ -5,31 +5,31 @@ pages:
...
    keywords: 'keyword1,keyword2'
-   description: 'page description'
+   description: 'Dragonfly Community'
  blog:
   # 博客列表页配置
   zh-cn:
    title: '博客'
    keywords: '关键词1，关键词2'
-   description: '页面内容简介'
+   description: 'Dragonfly的博客'
```
Comment (jim3ma) [inline docsite.config.yml:30]: Dragonfly博客

File: docsite.config.yml (hunk 1 above)
Comment (inductor) [inline docsite.config.yml:19]:
```suggestion
   description: 'Dragonfly社区'
```

File: docsite.config.yml (hunk 2 above)
Comment (inductor) [inline docsite.config.yml:30]:
```suggestion
   description: 'Dragonfly博客'
```

File: docsite.config.yml [hunk 3]
```diff
@@ -5,31 +5,31 @@ pages:
...
    keywords: 'keyword1,keyword2'
-   description: 'page description'
+   description: 'An Open-source P2P-based Image and File Distribution System'
  community:
   # 社区页配置
   zh-cn:
    title: '社区'
    keywords: '关键词1，关键词2'
-   description: '页面内容简介'
+   description: '<span class="x x-first x-last">Dragonfly社区</span>'
```
Comment (inductor) [inline docsite.config.yml:19]:
```suggestion
   description: Dragonfly社区'
```

**Comment by jim3ma** [comment]
@gaius-qi Please help review and merge this PR.

**Review by gaius-qi — APPROVED** [review]
LGTM

**Pull request merged**